export_snowflake/flattening.py

```diff
@@ -64,7 +64,7 @@ def flatten_schema(d, parent_key=None, sep='__', level=0, max_level=0):
         if len(list(g)) > 1:
             raise ValueError(f'Duplicate column name produced in schema: {k}')
 
-    return dict(sorted_items)
+    return dict(items)
```

Comment:
Before, the items were processed in row level, so we reorder the column sequence to alphabetical order.
Since we process the csv.gz files directly, we don't have to sort the items orders. 
The merge query will match the columns by names, so don't have to worry the mismatching here without sorting.